novawallet/fr.lproj/Localizable.strings

```diff
@@ -1768,3 +1768,20 @@
 "common.address.coppied" = "Adresse copiée";
 "common.networks" = "Réseaux";
 "assets.search.token.hint" = "Rechercher par token";
+"sec.time.units" = "sec";
+"swaps.execution.dont.close.app" = "Ne fermez pas l'application!";
+"swaps.details.exec.time" = "Temps d'exécution";
+"swaps.details.total.fee" = "Frais totaux";
+"swaps.label.transfer" = "Transfert";
+"swaps.label.swap" = "Échange";
+"swap.route.details.subtitle" = "La voie que prendra votre jeton à travers différents réseaux pour obtenir le jeton souhaité.";
+"swaps.details.route" = "Itinéraire";
+"swaps.execution.transfer.details" = "Transfert de %@ vers %@";
+"swaps.execution.swap.details" = "Échanger %@ contre %@ sur %@";
+"common.of" = "%@ de %@";
+"swaps.execution.swap.failure" = "Échec de l'opération n°%@ (%@)";
+"swaps.label.crosschain" = "Transfert inter-chaînes";
+"swap.intermediate.too.low.amount.to.stay.abow.ed.message" = "Lors de l'exécution de l'échange, le montant intermédiaire reçu est %@, ce qui est inférieur au solde minimum de %@. Essayez de spécifier un montant d'échange plus important.";
+"common.fee.amount.prefixed" = "Frais : %@";
+"common.not.enough.to.pay.fee.message" = "\"Vous n'avez pas assez de solde pour payer les frais de réseau de %@. Le solde actuel est %@";
```

Comment:
Looks like hallucinated prefix \". Probably it is due `don't` in English version. I would replace English version with `do not` and remove manually the prefix in the translations.

Reply:
fixed, also fixed slavic plurals not showing:
![Simulator Screenshot - iPhone 15 Pro - 2024-12-18 at 12 30 47](https://github.com/user-attachments/assets/9062646c-c373-444c-bfd9-89eb6ee21948)
![Simulator Screenshot - iPhone 15 Pro - 2024-12-18 at 12 45 26](https://github.com/user-attachments/assets/0e471c88-b721-4c63-907b-8efa9275bd32)